### PR TITLE
11234 Adjusting wording in overview-of-pipeline-runs 

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Rewords a line in modules/overview-of-pipeline-runs.adoc per QE & SME feedback.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by FedeAlonso — APPROVED** [review]

File: modules/overview-of-pipeline-runs.adoc [hunk 1]
```diff
@@ -25,7 +25,14 @@ You can use catch up runs to ensure your pipeline runs do not permanently fall b
...
 
-You can view further information about each artifact by clicking the artifact name in the list. You can also view or download the content of the artifact itself by clicking the artifact URI in the list. If your browser can display the artifact content, for example, if the artifact is plain text, HTML, or markdown, the content does not download, but is automatically displayed in a new browser tab. If your browser cannot display the artifact content, for example, if the artifact is a model, the artifact automatically downloads instead. To download an artifact which is displayed in a browser tab, right-click on the content and then click *Save as*. 
+You can view further information about each artifact by clicking the artifact name in the list. You can also view or download the content of artifacts that are stored in S3 by clicking the active artifact URI link in the list.
```
Comment (syaseen-rh) [inline modules/overview-of-pipeline-runs.adoc:28]: Should this be _S3 or S3-compatible object storage_? Unless it's restricted only to AWS S3.

File: modules/overview-of-pipeline-runs.adoc [hunk 2]
```diff
@@ -25,7 +25,14 @@ You can use catch up runs to ensure your pipeline runs do not permanently fall b
...
 
-You can view further information about each artifact by clicking the artifact name in the list. You can also view or download the content of the artifact itself by clicking the artifact URI in the list. If your browser can display the artifact content, for example, if the artifact is plain text, HTML, or markdown, the content does not download, but is automatically displayed in a new browser tab. If your browser cannot display the artifact content, for example, if the artifact is a model, the artifact automatically downloads instead. To download an artifact which is displayed in a browser tab, right-click on the content and then click *Save as*. 
+You can view further information about each artifact by clicking the artifact name in the list. You can also view or download the content of artifacts that are stored in S3 by clicking the active artifact URI link in the list. 
+
+[NOTE]
+====
+Artifacts that are not stored in S3 are not available to download, and will not appear with an active URI link.
```
Comment (syaseen-rh) [inline modules/overview-of-pipeline-runs.adoc:32]: Same note here

File: modules/overview-of-pipeline-runs.adoc (hunk 1 above)
Comment (HumairAK) [reply modules/overview-of-pipeline-runs.adoc:28]: S3-compatible is correct

Comment (grainnejenningsRH) [reply modules/overview-of-pipeline-runs.adoc]: Thanks both! Updated 👍

**Review by syaseen-rh — APPROVED** [review]

**Pull request merged**